src/js/components/artifacts/__snapshots__/releaseslist.test.js.snap

```diff
@@ -2,103 +2,98 @@
 
 exports[`ReleasesList Component renders correctly 1`] = `
 <div
-  className="repository-list flexbox column"
-  style={
-    Object {
-      "alignItems": "flex-start",
-    }
-  }
+  class="repository-list flexbox column"
```

Comment:
Something in these changes has affected the layout a bit – if you compare it to https://hosted.mender.io/ui/#/releases, you can see this left column is now being cut off a bit

Reply:
hmm that should not have changed with the changes in here (the snapshot-change is just because I had to render it differently to also see the selection menu...) - it might have been from the earlier PR that fixed the scrolling behaviour for the releases and devices list...

Reply:
You're right, sorry! I see this and couple of other small things on master too. Will follow up on those separately.